lib/nodes/addon/components/driver-pnap/component.js

```diff
@@ -0,0 +1,127 @@
+import { alias } from '@ember/object/computed';
+import { get, set } from '@ember/object';
+import Component from '@ember/component';
+import NodeDriver from 'shared/mixins/node-driver';
+import layout from './template';
+import { inject as service } from '@ember/service';
+
+
+const DRIVER = 'pnap';
+const CONFIG = 'pnapConfig';
+
+const LOCATION_CHOICES = [
```

Comment:
`LOCATION_CHOICES`, `OS_CHOICES`, `TYPE_CHOICES` all include a key/value pair but each of the selects that consume the respective arrays has `optionLabelPath` set to `value`. Since you're setting this option you can drop the name param from these arrays.